gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/TemplateMapperTest.java

```diff
@@ -184,26 +183,6 @@ public void When_MapTemplateWithoutSnomedCode_Expect_DegradedCode() {
         assertThat(parentObservation.getCode().getCodingFirstRep()).isEqualTo(DegradedCodeableConcepts.DEGRADED_OTHER);
     }
 
-    private void assertQuestionnaireResponse(QuestionnaireResponse questionnaireResponse, String encounter, String linkId,
-        String authored) {
-        assertThat(questionnaireResponse.getId()).isEqualTo(COMPOUND_ID + QUESTIONNAIRE_SUFFIX);
-        assertThat(questionnaireResponse.getMeta().getProfile().getFirst().getValue()).isEqualTo(QUESTIONNAIRE_META);
-        assertThat(questionnaireResponse.getIdentifier().getSystem()).isEqualTo(IDENTIFIER);
-        assertThat(questionnaireResponse.getIdentifier().getValue()).isEqualTo(COMPOUND_ID);
-        assertThat(questionnaireResponse.getParentFirstRep().getResource().getIdElement().getValue()).isEqualTo(COMPOUND_ID);
-        assertThat(questionnaireResponse.getStatus()).isEqualTo(QuestionnaireResponse.QuestionnaireResponseStatus.COMPLETED);
-        assertThat(questionnaireResponse.getSubject().getResource().getIdElement().getValue()).isEqualTo(PATIENT_ID);
-        assertThat(questionnaireResponse.getItemFirstRep().getLinkId()).isEqualTo(linkId);
-        assertThat(questionnaireResponse.getAuthoredElement().asStringValue()).isEqualTo(
-            DateFormatUtil.parseToDateTimeType(authored).asStringValue());
-
-        if (encounter == null) {
-            assertThat(questionnaireResponse.getContext().getResource()).isNull();
-        } else {
-            assertThat(questionnaireResponse.getContext().getResource().getIdElement().getValue()).isEqualTo(encounter);
-        }
-    }
-
```

Comment:
There are usages of this method, the code is just commented out. Removing the method, but not the commented invocations sounds like a big smell.